### ISSUE TITLE
Enable OpenGraph and Twitter Metadata to Improve Shareability of MAP pages on Social Media 

### ISSUE THREAD

**Issue by aeroplanejane** [bbc/simorgh#4753]
**Is your feature request related to a problem? Please describe.**
We need to pass the correct props to opengraph so information about the page is shared to social media platforms. 

**Describe the solution you'd like**
We need to ensure we are setting the correct opengraph and twitter card metadata to promote our content on social media. 

We should have feature parity with STYs and PGLs. 

The stories team have recently implemented this for PGLs: https://github.com/bbc/simorgh/issues/5158

We have a separate issue to enable to ensure the poster image is used on the `og:image` on MAPs 
See: https://github.com/bbc/simorgh/issues/4708

We can test and debug the opengraph metadata using this tool:  https://developers.facebook.com/tools/debug/

Generic:
- [ ] meta:description = Promo -> summary 

Open Graph (Facebook):
- [x] og:title = Promo -> Headline -> Headline
- [x] og:type = article
- [x] og:description = Promo -> summary 
- [x] og:site_name = Service -> name
- [x] og:locale = Service -> locale
- [x] og:url = Canonical URL of article
- [x] og:image = Promo -> IndexImage (same as promo image used on FP ).  If it doesn't exist, use Service -> Logo
- [x] og:image:alt = Promo -> IndexImage -> AltText. If it doesn't exist, use the Service -> Logo -> AltText
- [x] article:author = `https://www.facebook.com/bbcnews` across all services

Twitter:
- [x] twitter:creator = Service name e.g. `@BBCMundo`
- [x] twitter:site = Service name e.g. `@BBCMundo`
- [x] twitter:title = Promo -> Headline -> Headline
- [x] twitter:description = Promo -> Summary
- [x] twitter:img:src = Promo -> IndexImage (same as promo image used on FP ).  If it doesn't exist, use Service -> Logo
- [x] twitter:image:alt = Promo -> IndexImage -> AltText. If it doesn't exist, use the Service -> Logo -> AltText

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]
We can test and debug the opengraph metadata using this tool:  https://developers.facebook.com/tools/debug/
Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

**Comment by pharingee** [comment]
Open for testing

**Comment by LilyL0u** [comment]
What is 'FP'? In the sentence 'og:image = Promo -> IndexImage (same as promo image used on FP ). If it doesn't exist, use Service -> Logo'
@pharingee @aeroplanejane

**Comment by LilyL0u** [comment]
Also, I don't see 'meta:description' anywhere @pharingee @aeroplanejane

**Comment by aeroplanejane** [comment]
@LilyL0u FP is frontpage

**Comment by aeroplanejane** [comment]
Where don't you see meta:description? It should be there - so if it's not we need to add it. Can you send me the link you're testing @LilyL0u ? Thanks

**Comment by LilyL0u** [comment]
@aeroplanejane testing with https://www.bbc.com/pidgin/tori-51309348 I don't find it when I search for 'meta:description' in the html and I also don't see it using the facebook debug tool

**Comment by karinathomasbbc** [comment]
@LilyL0u I think the description of this issue is slightly incorrect, because we copied this from another issue. I think the intent was that we have a description field in the meta part of the HTML (which should already been there, and existed before we did this piece of work).

So, we can either remove 
> meta:description = Promo -> summary

above, or just verify that the HTML meta is correct?
